### ISSUE TITLE
Display the date of the newest available build on the download page

### ISSUE THREAD

**Issue by Ace-Dragon** [Calinou/godot-builds-ci#2]
What I mean by this is while the site mentions the builds being done nightly. It is not always the case.

An example, the latest Win64 build was done a few days ago on the 6th, but you wouldn't know whether or not the build updated until after you download it and unzip the contents (and then looking at the _last modified_ date next to the Godot binary).

In turn it would really help if the date the build was done was also mentioned (because like all automatic build algorithms, the build is not guaranteed to have the last day's commits 100 percent of the time).

**Comment by dodgyville** [comment]
Perhaps if the build date was part of the filename

**Comment by witten** [comment]
Yes, just ran into this myself today. The date somewhere would be really nice! Especially since I've found some [other sites](https://fixnum.org/godot/) with super out-of-date unofficial Godot Engine builds.

**Comment by Calinou** [comment]
I recently added JSON manifests for each build (`<file name>.manifest.json`), which include the commit, build date (currently being added) and SHA-256 hash. I'll try to find a way to expose this information on the web page soon :slightly_smiling_face:
